src/Flex.php

```diff
@@ -60,6 +60,7 @@
  */
 class Flex implements PluginInterface, EventSubscriberInterface
 {
+    public static $storedOperations = null;
```

Comment:
```suggestion
    public static $storedOperations = [];

```
(including the blank line after)